backend/src/BuiltinExecution/Libs/Math.fs

```diff
@@ -92,14 +93,15 @@ let fns : List<BuiltInFn> =
 
          This function is the inverse of <fn Math.cos>."
       fn =
+        let optType = VT.float
```

Comment:
Inlining would be more readable here too

Reply:
yeah I was on the fence about a few of these Option ones - will update in subsequent PR